packages/tests/scripts/pvt.json

```diff
@@ -98,10 +88,20 @@
       "remotedebug-chatdata-customapi-ts-openai-win-only",
       "remotedebug-chatdata-customapi-py-azureopenai-win-only",
       "remotedebug-chatdata-customapi-py-openai-win-only"
-      
-
     ],
-    "node-20": ["localdebug-obo-tab"]
+    "node-18": [
+      "localdebug-obo-tab",
```

Comment:
Should OBO tab be in the node-20 section instead?

Reply:
yes, obo has both node 18 and 20 testing
https://github.com/OfficeDev/teams-toolkit/blob/hellyzh/node20/packages/tests/scripts/pvt.json#L171
![image](https://github.com/user-attachments/assets/7cb92abc-e283-40d8-88e0-643d05fcfea7)